### PR TITLE
docs: update some links

### PR DESCRIPTION
**Pull request by CalMacCQ**
# Description

Please summarise the changes.

# Related issues

Please mention any github issues addressed by this PR.

# Checklist

- [ ] I have performed a self-review of my code.
- [ ] I have commented hard-to-understand parts of my code.
- [ ] I have made corresponding changes to the public API documentation.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have updated the changelog with any user-facing changes.

### CODE REVIEW

File: pytket/extensions/qiskit/tket_backend.py [hunk 1]
```diff
@@ -66,9 +66,8 @@ class TketBackend(BackendV2):
     :py:class:`qiskit.aqua.QuantumInstance`, providing a custom
     :py:class:`qiskit.transpiler.PassManager` with a
     :py:class:`qiskit.transpiler.passes.Unroller`. For examples, see the `user manual
-    <https://tket.quantinuum.com/user-manual/manual_backend.html#embedding-into-
-    qiskit>`_ or the `Qiskit integration example <ht
-    tps://github.com/CQCL/pytket/blob/main/examples/qiskit_integration. ipynb>`_.
+    <https://docs.quantinuum.com/tket/user-guide/manual/manual_backend.html#embedding-into-
```
Comment (CalMacCQ) [inline pytket/extensions/qiskit/tket_backend.py:69]: linting check is failing becuse therse lines are too long. Should we disable the linting for this line @cqc-melf

Comment (CalMacCQ) [reply pytket/extensions/qiskit/tket_backend.py:69]: These links were not clickable to begin with it seems.

Comment (cqc-melf) [reply pytket/extensions/qiskit/tket_backend.py:69]: I think it should be possible to have the links on multiple line with adding `\` at the end of the line? Have you tried that?

Comment (CalMacCQ) [reply pytket/extensions/qiskit/tket_backend.py:69]: I think just using a new line works. The link will not be clickable in the source file but sphinx produces a valid link in the html pages.

**Pull request closed without merging**